unepwcmc/action-agenda
# refactor: conditonally reduced footer

# Summary

- reduces footer on form pages
- fixes issue where going to [/commitments/new](http://localhost:3000/commitments/new) (with no criterium_id param) would result in a partial 404

# Testing

- go to [/criteria/new](http://localhost:3000/criteria/new) and check footer is as per [design](https://xd.adobe.com/view/85f38f01-ca5a-4c6b-a393-28b15a7e278b-d0a0/specs)
- complete form to get to commitments form and check footer is the same
- check footer includes full content on other pages
